.gitignore

```diff
@@ -0,0 +1,12 @@
+# OpenWakeWord
```

Comment:
perhaps we reference managed volumes instead to prevent this?

i.e.
volumes:
  openwakeword-data:
  whisper-data:
  piper-data:

this is what I did in my version. 
we could also add -gpu for volumes connected to gpu enabled instances in the GPU compose file so that we can keep data seperate between instance types.

Reply:
Do you mean non binded mounts? But then adding custom models (thinking mainly about OpenWakeWord here) is hard, with binded mounts you can just move the model to that directory. Also I don't think there will be a case where you want to move from GPU to NONGPU changing models, but probably I am wrong there.

Reply:
I think I agree with you here, probably the best way is to not bind them by default and then you can bind them extending the docker compose and point wherever you have the custom model.

Or maybe we could look at passing it as a parameter, haven't looked into it, I'm still fighting to generate the custom model actually.